VespaGAS.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
No, if `Volume` already does then it's ok. I was just elaborating on Dom's response.

I think your list of equality conditions just above is the correct way to implement equality.